File: Java/Count-1-in-Binary.java
public class Solution {
    /**
     * @param num: an integer
     * @return: an integer, the number of ones in num
     */
    public int countOnes(int num) {
        // write your code here
        int res = 0;
        while (num != 0) {
        	num = num & (num - 1);
        	res ++;
        }
        return res;
    }
};

File: Python/Count-1-in-Binary.py
class Solution:
    # @param num: an integer
    # @return: an integer, the number of ones in num
    def countOnes(self, num):
        # write your code here
        res = 0
        while num != 0:
            res += 1
            num = num & (num - 1)
        return res